espanol/barracas/llave/hernandez.md

```diff
@@ -1,3 +1,7 @@
 El capitán Hernández sin hacer un mínimo gesto de alteración, sacó una llave de su bolsillo y se la pasó a Beatito.
 
--Vé y entregasela al Consejero- le ordenó.
\ No newline at end of file
+-Vé y entregasela al Consejero- le ordenó.
+
+[Beatito pensativo, toda urgencia olvidada, salió de la habitación y se apoyó contra la puerta cerrada, dudando.](duda/renunciar.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.